files/podman.conf

```diff
@@ -0,0 +1 @@
+net.ipv4.ping_group_range=0 2000000
```

Comment:
I don't see a requirement for enabling the unprivileged ping at this moment, so I'd suggest not including this until we know we actually need it. And this configuration is not specific to podman, a more meaningful name like `allow_unprivileged_ping.conf` could be better.